scripts/events_module/handle_short_events.py

```diff
@@ -562,6 +586,14 @@ def handle_death_history(self):
                             self.random_cat,
                         )
 
+                    if cat.status == "leader":
+                        self.current_lives -= 1
+                        if self.current_lives != game.clan.leader_lives:
+                            while self.current_lives > game.clan.leader_lives:
+                                History.add_death(
+                                    cat, "multi_lives", other_cat=self.random_cat
```

Comment:
```suggestion
                                    cat, "multi_lives"
```
Possibly meant to be this Leaving as suggestion until someone who understands the death event code can chip in